client-metrics/src/stakerConfig.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  ExecutionClientMainnet,
+  ConsensusClientMainnet,
+  ExecutionClientGnosis,
+  ExecutionClientPrater,
+  ConsensusClientPrater,
+  ConsensusClientGnosis,
+
+} from "@dappnode/types";
+
+export function convertEnvsToURL() {
+  const executionClientMainnet = process.env._DAPPNODE_GLOBAL_EXECUTION_CLIENT_MAINNET as ExecutionClientMainnet;
+  const ConsensusClientMainnet = process.env._DAPPNODE_GLOBAL_CONSENSUS_CLIENT_MAINNET as ConsensusClientMainnet;
+  const executionClientPrater = process.env._DAPPNODE_GLOBAL_EXECUTION_CLIENT_PRATER as ExecutionClientPrater;
+  const ConsensusClientPrater = process.env._DAPPNODE_GLOBAL_CONSENSUS_CLIENT_PRATER as ConsensusClientPrater;
+  const executionClientGnosis = process.env._DAPPNODE_GLOBAL_EXECUTION_CLIENT_GNOSIS as ExecutionClientGnosis;
+  const ConsensusClientGnosis = process.env._DAPPNODE_GLOBAL_CONSENSUS_CLIENT_GNOSIS as ConsensusClientGnosis;
+
+  let executionClientMainnetUrl: string,
+    executionClientPraterUrl: string,
+    executionClientGnosisUrl: string,
+    consensusClientMainnetUrl: string,
+    consensusClientPraterUrl: string,
+    consensusClientGnosisUrl: string;
+
+  switch(executionClientMainnet) {
```

Comment:
This code should not be placed here but in the npm dappnode package @dappnode/types.

Could you open an issue in that repository (something like "create utility function `getExecutionEndpointFromDnpName`" and same for consensus) and leave a coment here refereing the issue please

Reply:
https://github.com/dappnode/types/issues/17